Healthy/Classes/Modules/Splash/SplashViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+final class SplashViewController: UIViewController {
+
+    // MARK: Outlets
+
+    @IBOutlet private (set) weak var startCookingButton: UIButton!
+    @IBOutlet private (set) weak var logoCaptionLabel: UILabel!
+    @IBOutlet private (set) weak var headerTitleLabel: UILabel!
+    @IBOutlet private (set) weak var headerCaptionLabel: UILabel!
+
+    // MARK: Properties
+
+    private let viewModel: SplashViewModelType
+
+    // MARK: Init
+
+    init(viewModel: SplashViewModelType) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureAppearance()
+    }
+}
+
+// MARK: - Actions
+
+private extension SplashViewController {
+    @IBAction func didTapStartCooking(_ sender: Any) {
+        if let login = Bundle.main.loadNibNamed("LoginViewController", owner: self, options: nil)?.first as? LoginViewController {
+            self.navigationController?.pushViewController(login, animated: true)
+        }
```

Comment:
```suggestion
        // TODO: Replace with a coordinator 
        let viewModel = LoginViewModel()
        let viewController = LoginViewController(viewModel: viewModel)
        navigationController?.pushViewController(viewController, animated: true)
```

Reply:
I used this code before it is not working as we need